thorin-config.cmake

```diff
@@ -41,6 +42,7 @@ FIND_PATH ( THORIN_CMAKE_DIR
     NAMES
         ThorinRuntime.cmake
     PATHS
+        ${THORIN_CMAKE_DIR}
```

Comment:
This is a deprecated commit anyway and not in the final change set. Please refer to https://github.com/AnyDSL/thorin/pull/64/files for the actual changes of the pull request.